website/pages/en/network/developing.mdx

```diff
@@ -2,52 +2,104 @@
 title: Developing
 ---
 
-Developers are the demand side of The Graph ecosystem. Developers build subgraphs and publish them to The Graph Network. Then, they query live subgraphs with GraphQL in order to power their applications.
+Learn developer fundamentals and review the basics of a subgraph’s lifecycle. To start coding right away and build on The Graph, go to [Developer Quick Start](/quick-start/)
 
-## Subgraph Lifecycle
+## Overview
 
-Subgraphs deployed to the network have a defined lifecycle.
+As a developer, you need important data to build and power your dapps. Querying and indexing that blockchain data can be challenging, but The Graph provides a solution to this issue.
 
-### Build locally
+On The Graph, you can:
 
-As with all subgraph development, it starts with local development and testing. Developers can use the same local setup whether they are building for The Graph Network or a local Graph Node, leveraging `graph-cli` and `graph-ts` to build their subgraph. Developers are encouraged to use tools such as [Matchstick](https://github.com/LimeChain/matchstick) for unit testing to improve the robustness of their subgraphs.
+1.  Create, deploy and publish subgraphs to The Graph using **Graph CLI** and [Subgraph Studio](https://thegraph.com/studio/).
+2.  Use GraphQL to query live subgraphs and power dapps.
 
-> There are certain constraints on The Graph Network, in terms of feature and network support. Only subgraphs on [supported networks](/developing/supported-networks) will earn indexing rewards, and subgraphs which fetch data from IPFS are also not eligible.
+### What is GraphQL?
 
-### Deploy to Subgraph Studio
+- [GraphQL](/querying/graphql-api/)is the query language for APIs and a runtime for executing those queries with your existing data. The Graph uses GraphQL to query subgraphs.
```

Comment:
```suggestion
[GraphQL](/querying/graphql-api/) is a declarative query language for APIs and a runtime for executing those queries. The Graph uses GraphQL to query subgraphs, ensuring you receive exactly the data you need—nothing more, nothing less.
```
Or something like that, but basically I feel like we should at least mention the difference between GraphQL and other types of APIs (e.g. REST) in a very high-level way.

Reply:
Also, I'm not sure our [/querying/graphql-api](https://thegraph.com/docs/en/querying/graphql-api/) page is a good introduction to GraphQL itself, it's more about how The Graph uses it. I wonder if we should link to https://graphql.org instead.

Reply:
we can link to https://graphql.org/learn/